tensorflow/python/keras/callbacks.py

```diff
@@ -443,6 +449,38 @@ class Callback(object):
           the number of samples in the current batch.
       on_batch_end: logs include `loss`, and optionally `acc`
           (if accuracy monitoring is enabled).
+    
+  Example:
+  ```python  
+  # Import the datetime module to display the time
+  import datetime
+  
+  # A custom callback displaying the loss and accuracy 
+  # from the `logs` dict at the end of a batch/epoch
+  class MyLossAccuracyCallback(tf.keras.callbacks.Callback):
```

Comment:
https://www.tensorflow.org/beta/guide/keras/custom_callback has a similar example. Can we make the input, code and class name in this example be consistent with that?

Reply:
@rchao Absolutely. See below. Thank you!

Small changes to that example from the Keras Custom Callback in TF 2 Beta [guide](https://www.tensorflow.org/beta/guide/keras/custom_callback): 

1) `x_train` and `y_train` -> `data` and `labels` for consistency, so it's just like most callback examples in `callbacks.py`, unless you want to use the "classic" `x` and `y` notation. 

2) Also, it's probably good to still define `steps_per_epoch` - as before - as an iteration over training data i.e. length of training data (`x_train` or `data`) divided by `batch_size` (See this [TF 2 Beta tutorial of TF Hub with Keras](https://www.tensorflow.org/beta/tutorials/images/hub_with_keras) which uses the same approach).

```python
# A custom callback displaying the loss and mean absolute error 
# from the `logs` dict at the end of a batch/epoch
class LossAndErrorPrintingCallback(tf.keras.callbacks.Callback):

  def on_train_batch_end(self, batch, logs=None):
    print('For batch {}, loss is {:7.2f}.'.format(batch, logs['loss']))

  def on_test_batch_end(self, batch, logs=None):
    print('For batch {}, loss is {:7.2f}.'.format(batch, logs['loss']))

  def on_epoch_end(self, epoch, logs=None):
    print('The average loss for epoch {} is {:7.2f} and mean absolute error is {:7.2f}.'.format(epoch, logs['loss'], logs['mae']))

batch_size=64
epochs=3

model.fit(data, labels,
          batch_size=batch_size,
          steps_per_epoch=int(np.ceil(data.shape[0])/batch_size),
          epochs=epochs,
          verbose=0,
          callbacks=[LossAndErrorPrintingCallback()])
```

Reply:
@rchao @rthadur if there are further improvements needed to be done let me know. thanks for the help 👍